src/main/java/org/jabref/gui/entryeditor/FileDragDropPreferences.java

```diff
@@ -0,0 +1,39 @@
+package org.jabref.gui.entryeditor;
+
+public class FileDragDropPreferences {
+
+    private boolean copyFile;
+    private boolean linkFile;
+    private boolean renameCopyFile;
+
+    public FileDragDropPreferences(boolean copyFile, boolean linkFile, boolean renameCopyFile) {
```

Comment:
At this point, it appears to be cleaner to introduce an enum for the three different behaviors, especially since they are mutually exclusive.

Reply:
Switched to enum:
https://github.com/JabRef/jabref/pull/4408/commits/66443c00d4d49c4f7319b6d24ff17a20df8ddd22#diff-65271fca5ef3c430d80bc31ef69fdf56R3